ui/src/app/globals.css

```diff
@@ -11,3 +11,23 @@
         font-family: var(--font-source-sans-3), 'Helvetica', Arial, sans-serif;
     }
 }
+
+.variant-default {
+    color: #fff;
+    border-width: 1px;
+    border-style: solid;
+    border-color: rgba(0, 0, 0, 0.1) rgba(0, 0, 0, 0.1) rgba(0, 0, 0, 0.25);
+    text-shadow: 0 1px 1px #444;
+    background-color: #6fa9be;
+    background: -moz-linear-gradient(to bottom, #7db1c4, #5a9cb4);
+    background: -webkit-gradient(linear, left top, left bottom, from(#7db1c4), to(#5a9cb4));
+    background: -webkit-linear-gradient(to bottom, #7db1c4, #5a9cb4);
+    background: -o-linear-gradient(to bottom, #7db1c4, #5a9cb4);
+    background: -ms-linear-gradient(to bottom, #7db1c4, #5a9cb4);
+    background-repeat: repeat-x;
+    transition: background 0.15s ease-in-out;
+}
+
+.variant-default:hover {
+    background: #004e59;
+}
```

Comment:
Would it be possible to keep everything else but the `background` and `transition` CSS as Tailwind CSS classes? Then we should rename `.variant-default` to describe this CSS class better. Maybe `.uh-groupings-button-gradient`.

Reply:
Also it seems that the best practice is to use `@layer components` :mhttps://tailwindcss.com/docs/adding-custom-styles#adding-component-classes